src/components/ScreenSplit.jsx

```diff
@@ -5,48 +5,46 @@ import { makeStyles } from "@material-ui/core/styles";
 const useStyles = makeStyles((theme) => ({
   gridItem: {
     [theme.breakpoints.up("xs")]: {
-      marginTop: "20%",
-      marginBottom: "20%"
+      marginBottom: "-130%",
```

Comment:
Can we do it differently? -130% margin seems wrong

Reply:
minus margin is a legit way. but i fixed a better solution i hope 👍